### ISSUE TITLE
Additional user config

### ISSUE THREAD

**Issue by MrJoelKelly** [mike-north/ember-intercom-io#141]
I've attempted to add additional user config details to pass to intercom by setting additional params in the service and initializing them in our config. When I `console.log` the intercom services user values, these values appear to be set, but none seem to be passed to Intercom.

I saw that PR #30 was supposed to add this ability, but it was not added to the docs, and it doesn't seem immediately clear to me how we are supposed to add custom attributes. Please could someone advise how we can do this, preferably with a short example?

Thank you!

**Comment by GSmes** [comment]
@MrJoelKelly Have you been able to figure this out? I know it's only been a week, but I'm having a similar issue.

_Code example:_
```javascript
get(this, 'intercom').set('user.email', 'email3@example.com')
get(this, 'intercom').set('user.name', 'Michael Jordan')
get(this, 'intercom').set('user.id', '1010101')
get(this, 'intercom').set('user.custom', 'test')
```

Using the example above, the first 3 properties get set in Intercom (`email`, `name`, and `user_id`), but the final property does not. I have tried several variations of custom attributes, including:
- using snake-case for the key (e.g., `custom_prop`)
- using camel-case for the key (e.g., `customProp`)
- providing a custom attribute that we've created on Intercom users before (e.g., `user.type`)
- providing a custom attribute that has never been created on an Intercom user before (e.g., `user.admin_url` and `user.adminUrl`)
